scripts/create_summary.py

```diff
@@ -0,0 +1,62 @@
+import os
```

Comment:
Could you add a shebang such as the following?

```
#!/usr/bin/env bash
```

Reply:
Done.